devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -0,0 +1,132 @@
+ *
+ */
+
+metadata {
+	definition (name: "Z-Wave Basic Window Shade", namespace: "smartthings", author: "SmartThings") {
+		capability "Stateless Curtain Power Button"
+		capability "Configuration"
+		capability "Actuator"
+		capability "Health Check"
+
+		command "open"
+		command "close"
+		command "pause"
+
+		fingerprint mfr:"0086", prod:"0003", model:"008D", deviceJoinName: "Aeotec Nano Shutter"
+		fingerprint mfr:"0086", prod:"0103", model:"008D", deviceJoinName: "Aeotec Nano Shutter"
+	}
+
+	tiles(scale: 2) {
+ 		standardTile("open", "device.open", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label:'Open', action:"open"
+		}
+		standardTile("close", "device.close", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label:'Close', action:"close"
+		}
+		standardTile("pause", "device.pause", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label:'Pause', action:"pause"
+		}
+
+		details(["open","close","pause"])
+	}
+}
+
+def parse(String description) {
+	def result = []
+	if (description.startsWith("Err")) {
+		result = createEvent(descriptionText:description, isStateChange:true)
+	} else {
+		def cmd = zwave.parse(description)
+		if (cmd) {
+			result += zwaveEvent(cmd)
+		}
+	}
+	log.debug "Parse returned: ${result}"
+	result
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	log.debug "Security Message Encap ${cmd}"
+	def encapsulatedCommand = cmd.encapsulatedCommand()
+	if (encapsulatedCommand) {
+		zwaveEvent(encapsulatedCommand)
+	} else {
+		log.warn "Unable to extract encapsulated cmd from $cmd"
+		createEvent(descriptionText: cmd.toString())
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
+	state.shadeState = cmd.value ? "closing" : "opening"
+}
+
+def zwaveEvent(physicalgraph.zwave.Command cmd) {
+	log.warn "Unhandled ${cmd}"
+	createEvent(descriptionText: "An event came in")
+}
+
+def setButton(button) {
+	switch(button) {
+		case "open":
+			open()
+			break
+		case "close":
+			close()
+			break
+		default:
+			pause()
+			break
+	}
+}
+
+def open() {
+	state.shadeState = "opening"
+	secure(zwave.basicV1.basicSet(value: 0x00))
+}
+
+def close() {
+	state.shadeState = "closing"
+	secure(zwave.basicV1.basicSet(value: 0xFF))
+}
+
+def pause() {
+	def value = state.shadeState == "opening" ? 0xFF : 0x00
+	def result = state.shadeState != "paused" ? secure(zwave.switchBinaryV1.switchBinarySet(switchValue: value)) : []
+	state.shadeState = "paused"
+	result
+}
+
+def ping() {
+	secure(zwave.switchMultilevelV3.switchMultilevelGet())
+}
+
+def installed() {
+	log.debug "Installed ${device.displayName}"
+	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	sendEvent(name: "availableCurtainPowerButtons", value: ["open", "close", "pause"])
```

Comment:
I _think_ you just need to make this JSON

Reply:
👀 @PKacprowiczS

Reply:
so based on your comment on the ticket, this _doesn't_ need to be JSON?

Reply:
No, it doesn't.

Reply:
The capability is defined as JSON:

```
  availableCurtainPowerButtons:
    schema:
      type: object
      additionalProperties: false
      properties:
        value:
          type: array
          items:
            type: string
            enum:
              - open
              - close
              - pause
    type: JSON_OBJECT
```
I'd try it as JSON and assuming that works then change it to that. If it doesn't,  there is a bug somewhere. But it should be a JSON array containing items of type string.

Reply:
It works for both JSON and list.